editor/inspector/assets/fbx/model.js

```diff
@@ -656,7 +786,102 @@ const Elements = {
+                        break;
+                }
+            });
+            // Listening to the addition and removal of the lod hierarchy
+            panel.$.lodItems.addEventListener('click', (event) => {
+                event.stopPropagation();
+                event.preventDefault();
+                const path = event.target.getAttribute('path');
+                const index = Number(event.target.getAttribute('key'));
+                if (path === 'insertLod') {
+                    if (Object.keys(panel.meta.userData.lods.options).length >= 8) {
+                        console.warn('Maximum 8 LOD, Can\'t add more LOD');
+                        return;
+                    }
+                    const preScreenRatio = panel.meta.userData.lods.options[index].screenRatio;
+                    const nextScreenRatio = panel.meta.userData.lods.options[index + 1] ? panel.meta.userData.lods.options[index + 1].screenRatio : 0;
+                    const preFaceCount = panel.meta.userData.lods.options[index].faceCount;
+                    const nextFaceCount = panel.meta.userData.lods.options[index + 1] ? panel.meta.userData.lods.options[index + 1].faceCount : 0;
+                    const option = {
+                        screenRatio: (preScreenRatio + nextScreenRatio) / 2,
+                        triangleCount: 0,
+                        faceCount: (preFaceCount + nextFaceCount) / 2,
+                    };
+                    // Insert the specified lod level
+                    for (let key = Object.keys(panel.meta.userData.lods.options).length - 1; key > index; key--) {
+                        panel.meta.userData.lods.options[parseInt(key) + 1] = panel.meta.userData.lods.options[key];
+                    }
+                    panel.meta.userData.lods.options[index + 1] = option;
+                    // update panel
+                    panel.$.lodItems.innerHTML = getLodItemHTML(panel.meta.userData.lods.options, panel.meta.userData.lods.isBuiltin);
+                    panel.dispatch('change');
+                } else if (path === 'deleteLod') {
+                    if (Object.keys(panel.meta.userData.lods.options).length <= 1) {
+                        console.warn('At least one LOD, Can\'t delete any more');
+                        return;
+                    }
+                    // Delete the specified lod level
+                    for (let key = index; key < Object.keys(panel.meta.userData.lods.options).length; key++) {
+                        panel.meta.userData.lods.options[key] = panel.meta.userData.lods.options[key + 1];
+                    }
+                    delete panel.meta.userData.lods.options[Object.keys(panel.meta.userData.lods.options).length - 1];
+                    // update panel
+                    panel.$.lodItems.innerHTML = getLodItemHTML(panel.meta.userData.lods.options, panel.meta.userData.lods.isBuiltin);
+                    panel.dispatch('change');
```

Comment:
this method call from click , so after dispatch chane , need add `panel.dispatch('snapshot');`

Reply:
You are absolutely right